subprojects/base-services/src/testFixtures/groovy/org/gradle/internal/hash/TestFileHasher.groovy

```diff
@@ -17,7 +17,6 @@
 package org.gradle.internal.hash
 
 import com.google.common.io.Files
-import org.gradle.api.UncheckedIOException
```

Comment:
Was this an unused import? How is it possible that codenarc didn't report it?

Reply:
no, I've changed it to the JDK 8 `UncheckedIOException`, which is included by [Groovy by default](https://groovy-lang.org/differences.html#_default_imports)

Reply:
Ah, I was looking at `TestFile` instead of `TestFileHasher` 🤦‍♂ 
That's why I didn't find the exception in the class....